src/main/sensors/gyro.h

```diff
@@ -71,6 +71,11 @@ typedef struct gyroCalibration_s {
     int32_t cyclesRemaining;
 } gyroCalibration_t;
 
+// The gyro buffer is split 50/50, the first half for the transmit buffer, the second half for the receive buffer
+// This buffer is large enough for the gyros currently supported in accgyro_mpu.c but should be reviewed id other
+// gyro types are supported with SPI DMA.
+#define GYRO_BUF_SIZE 32
```

Comment:
Where is this used outside of `gyro_init.c`?

Reply:
At one point in accgyro_mpu.c, but no longer. Moved.